open_base/Velocity input
---
geometry_msgs/Pose2D output
